src/rules/noDuplicateVariableRule.ts

```diff
@@ -75,6 +80,12 @@ class NoDuplicateVariableWalker extends Lint.ScopeAwareRuleWalker<ScopeInfo> {
             }
         } else {
```

Comment:
now that I think about it, I think these branches should be flipped around. I always prefer to have the positive conditional first, and the negative branch in the `else`